tests/util.spec.coffee

```diff
@@ -117,3 +121,35 @@ describe 'Pine option merging', ->
 		m.chai.expect(
 			-> mergePineOptions(unknownKey: 'value', {})
 		).not.to.throw()
+
+describe 'ImgMakerHelper', ->
+
+	ROOT_URL = 'https://img.resin.io'
+
+	beforeEach =>
+		@requestStub = send: m.sinon.stub().returns(new Promise(
+			-> # never return, don't worry about responses
```

Comment:
Success! I think... Seems hound's `no_empty_functions` default is different to our local one for some reason. No idea why, but explicitly turning it off seems to have worked. Also fixed incorrect config key in `.hound.yml`. It now matches the docs, although I'm not wholly sure that's a major problem. Sorted anyway, and I've tested that it does work for other errors. Sorry for the notification noise!